tests/Mapper/ProgrammesDbToDomain/MasterBrandMapperTest.php

```diff
@@ -170,6 +170,35 @@ public function testGetDomainModelWithSetCompetitionWarning()
         $this->assertEquals($expectedEntity, $this->getMapper()->getDomainModel($dbEntityArray));
     }
 
+    public function testGetDomainModelWithEmbargoedCompetitionWarning()
+    {
+        $networkDbEntity = ['nid' => 'bbc_one'];
+        $versionDbEntity = ['pid' => 'p01m5mss', 'programmeItem' => null];
+
+        $this->setupNetworkMapper($networkDbEntity, $this->mockNetwork);
+
+        $this->mockVersionMapper->expects($this->never())
+            ->method('getDomainModel');
+
+        $dbEntityArray = [
```

Comment:
Are you sure you won't have a cleaner test by putting these stuff in a dataProvider?